src/rt/model.h

```diff
@@ -55,8 +68,14 @@ rt_signal_t *find_signal(rt_scope_t *scope, tree_t decl);
 rt_proc_t *find_proc(rt_scope_t *scope, tree_t proc);
 
 const void *signal_value(rt_signal_t *s);
+static inline const uint8_t *signal_value8(rt_signal_t *s)
```

Comment:
Any reason to add this rather than just casting the result of `signal_value()`?

Reply:
OK, but I'd prefer it renamed to `signal_value_u8` and moved to the end of the file (after the list of declarations).